crates/primitives/src/constants/mod.rs

```diff
@@ -121,6 +121,9 @@ pub const ETH_TO_WEI: u128 = FINNEY_TO_WEI * 1000;
 /// Multiplier for converting mgas to gas.
 pub const MGAS_TO_GAS: u64 = 1_000_000u64;
 
+/// Multiplier for blob size
+pub const BLOB_SIZE_BYTES: u64 = 128 * 1024;
```

Comment:
unclear where this is coming from

Reply:
From the EIP-4844 spec.  Relocated next to 1558 constants and renamed to match the spec.